test/helpers/PlayerInteractionWrapper.js

```diff
@@ -115,6 +115,9 @@ class PlayerInteractionWrapper {
                 });
             }
         } else {
+            if (leaderOptions.deployed === false) {
+                leaderCard.undeploy();
```

Comment:
Can we make this idempotent? Currently it will break if the user sets `deployed: false` when the leader is already undeployed

Reply:
Changed. Let me know if I did it right.

Reply:
Looks perfect! Thanks